### DISCUSSION TITLE
Getting transpiler configuration

### DISCUSSION THREAD

**Discussion by stephencranedesign** [vitest-dev/vitest#5419] [Q&A]
I'm wondering if there's anyway to get at the esbuild transpiler configuration used in vitest. The app I'm on is considering switching to vitest for our unit test runner but for now will be staying on webpack. It would be nice to keep the same transpiler configuration between our test/dev/build environments.

**Comment by AriPerkkio** [comment]
See https://vitejs.dev/config/shared-options.html#esbuild.
